client/Makefile

```diff
@@ -14,6 +20,57 @@ install:
 build:
 	npm run build
 
+
+# Development convenience methods
+
+.PHONY: server-requirements
+server-requirements:
+	virtualenv -p python3.6 ../venv
```

Comment:
When running `backend-dev` I get the error message:
> virtualenv -p python3.6 ../venv
The path python3.6 (from --python=python3.6) does not exist

Do I need to downgrade from python 3.7 to 3.6 or is this a different issue?

Reply:
different issue.  You can safely keep using Py 3.6, 3.7 or 3.8.

However, if you do use these Makefile targets, you will need to have python 3.6 installed.

Reply:
Good catch. Fixing this.